__test__/e2e/invite_texter.test.js

```diff
@@ -0,0 +1,42 @@
+import { selenium } from './util/helpers'
+import STRINGS from './data/strings'
+import { login, invite, people } from './page-functions/index'
+
+// Instantiate browser(s)
+const driver = selenium.buildDriver()
+const driverTexter = selenium.buildDriver()
+
+describe('Invite Texter workflow', () => {
+  const CAMPAIGN = STRINGS.campaigns.existingTexter
+  beforeAll(() => {
+    global.e2e = {}
+  })
+  afterAll(async () => {
+    await selenium.quitDriver(driver)
+    await selenium.quitDriver(driverTexter)
+  })
+
+  describe('(As Admin) Log In an admin to Spoke', () => {
+    login.tryLoginThenSignUp(driver, CAMPAIGN.admin)
+  })
+
+  describe('(As Admin) Create a New Organization / Team', () => {
+    invite.createOrg(driver, STRINGS.org)
+  })
+
+  describe('(As Admin) Invite a new User', () => {
+    people.invite(driver)
+  })
+
+  describe('(As Texter) Follow the Invite URL', () => {
+    describe('Create New Texter in Spoke', () => {
```

Comment:
I think this step is unnecessary, using this link i believe creates the texter account as an admin (since hitting /login in SUPPRESS_SELF_INVITE false mode autogenerates an admin/org invite). Using the second join link (from the campaign or even people menu) will create a texter (with texter permissions)